### ISSUE TITLE
Indicator on suggested actions to show that the action will open a URL

### ISSUE THREAD

**Issue by vzv111** [microsoft/BotFramework-WebChat#3833]
For the suggestion actions for adaptive cards, I would like to indicate what type of action would be taken by clicking the button. For example for "Action.CardType", I would like to indicate on the button that this will take you to an external URL:

![image](https://user-images.githubusercontent.com/42174202/113788374-b07ce200-9734-11eb-8d3a-32b8725a0811.png)

At the moment this is no possible as every button is rendered with the same CSS so there is no way to differentiate which type of button has been rendered.

Is it possible to include the action type as part of the CSS on the rendered button as shown below?

![image](https://user-images.githubusercontent.com/42174202/113788684-75c77980-9735-11eb-9497-c8b3bb302b75.png)

**Comment by corinagum** [comment]
Based on your HTML and image, this is not a suggested action (botbuilder type) but an Adaptive Card. If so, your request would need to be filed with the [Adaptive Cards team](https://github.com/microsoft/AdaptiveCards).

However, you are correct in that Suggested Actions _also_ do not indicate external openUrls, so that will need to be updated on Web Chat as well. I will update the original comment to reflect this.

**Comment by vzv111** [comment]
Thanks @corinagum - I've also posted the same request on Adaptive Cards repo too, I was using a Hero Card in the example I above.

**Comment by corinagum** [comment]
@vzv111 Makes sense! Herocards are styled after Adaptive Cards in Web Chat, which is why you see the `.ac-*` classnames.

**Comment by corinagum** [comment]
Moving this to our backlog
